README.md

```diff
@@ -1,4 +1,8 @@
 # webidl.js
+[![Build Status](https://travis-ci.org/extensibleweb/webidl.js.svg?branch=master&maxAge=2592)](https://travis-ci.org/extensibleweb/webidl.js)
+[![NPM version](https://badge.fury.io/js/webidl.svg?maxAge=2592)](http://badge.fury.io/js/webidl)
+[![Dependency Status](https://img.shields.io/david/dev/extensibleweb/webidl.js.svg?maxAge=2592)](https://david-dm.org/extensibleweb/webidl.js)
```

Comment:
[shields.io](http://shields.io) is a free service that serves repo badges for GitHub, npm, ruby, and many other projects.